server/transport.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/go-kit/kit/endpoint"
 	kitlog "github.com/go-kit/kit/log"
 	kithttp "github.com/go-kit/kit/transport/http"
-	"golang.org/x/net/context"
+	"context"
```

Comment:
Would it be better if something like `semver` were incorporated to manage all the versioning constraints?